test/stubs/stubPersonoversikt.ts

```diff
@@ -0,0 +1,15 @@
+import nock from 'nock';
+import { SYFOOVERSIKTSRVREST_ROOT } from '../../src/utils/apiUrlUtil';
+import aktivEnhetMockData from '../../Mock/Data/aktivEnhet.json';
```

Comment:
Hvorfor velger du å "gå tilbake" til .json-filer fra javascript? Ser ikke ut som om denne filen er i bruk dag og kunne vært slettet. Ser forsåvidt ut som det gjelder for noen eller alle av de andre mock-json-filene også.

Json-filer for mocking har i hvert fall foreløpig vært overflødig når man heller kan bruke .js for å få dynamiske data i stedet for statiske.

Reply:
Tja, tenkte ikke så nøye over det igrunn. Tenkte vel kanskje mest at det ble mest "reelt" da response fra server kommer som JSON. Men kan godt endre til typescript, så blir det likt med de andre appene våre